crates/oxc_linter/src/rules/eslint/no_useless_escape.rs

```diff
@@ -424,6 +627,43 @@ fn test() {
         r"var foo = /\（([^\）\（]+)\）$|\(([^\)\)]+)\)$/;",
         r#"var stringLiteralWithNextLine = "line 1\line 2";"#,
         r"var stringLiteralWithNextLine = `line 1\line 2`;",
+        r#""use\ strict";"#,
```

Comment:
Can you please add some of these cases to the expected fixes tests?

Reply:
Thank you for the reminder! I meant to do this before putting this up, I will add these test cases